fe/fe-core/src/main/java/com/starrocks/connector/jdbc/MysqlSchemaResolver.java

```diff
@@ -104,4 +110,81 @@ public Type convertColumnType(int dataType, String typeName, int columnSize, int
             return ScalarType.createUnifiedDecimalType(precision, max(digits, 0));
         }
     }
+
+    @Override
+    public List<String> listPartitionNames(Connection connection, String databaseName, String tableName) {
+        String partitionNamesQuery = "select PARTITION_DESCRIPTION from INFORMATION_SCHEMA.PARTITIONS WHERE TABLE_SCHEMA = ? " +
+                "AND TABLE_NAME = ? AND PARTITION_NAME IS NOT NULL";
```

Comment:
Wonderful! It's a great work. 

Can I ask some question here。Mysql supports different partition types: RANGE/LIST/HASH/SubPartitions. do here we support all partition types?

Reply:
Due to limitations in other implementations of dependencies, currently only range partitioning is supported, with partition fields supporting int and date. Currently, sub partitioning is not supported. I have considered this matter and feel that sub partitioning is currently of little significance for OLAP.

Reply:
THX. I agree with you. Just need to know what's the limitation about jdbc catalog.

Reply:
ok，I heard that in version 3.1.1, mv will support list partitioning, and I will follow the community to add the list partitioning capability of the jdbc catalog